osmhelper/osm_stops.py

```diff
@@ -1,11 +1,7 @@
 # coding=utf-8
 
-import overpy
 import re
-import sys
 from math import cos, sin, atan2, sqrt, radians, degrees
-from transitfeed import util
-
 
 STOP_REGEX = re.compile('(TICAN|TISAN|TICEN|TITRI|TILAG|TIRIO|TISAC).*')
```

Comment:
Hmm, but `normalize_name` is only getting called from `match_first_stops` which only get's called from `TripsCreatorFenix` when everything is already build as it should be. If we start working on this, it would already be related very much to #30... And I wanted to make this PR small :laughing: